src/utils/validateEntity/index.js

```diff
@@ -0,0 +1,36 @@
+const { query: Hasura } = require('../hasura');
+const { findIdea, findProject, findThought } = require('./queries/queries');
+
+const identifierMap = {
```

Comment:
Can we switch to using DB & maintaining a cache just like we are doing in [getActivityId](https://github.com/Shivz3232/inovact-nodejs-veyga/blob/409034ef7df2d8063d6c36ab79b9dd4f11418350/src/utils/getActivityId/index.js#L4) ?

Reply:
Instead of hardcoding...